### PR TITLE
Implement style updates on mobile nav

### PR DESCRIPTION
**Pull request by danahertzberg**

### CODE REVIEW

**Comment by joeparsons** [comment]
The GitHub Actions failure is being caused by the following HTML validation issues reported by v.Nu HTML Checker (in the compiled documentation HTML):
```
"file:/azbuild/arizona-bootstrap/_site/docs/2.0/components/navbar-off-canvas/index.html":475.7-475.74: error: Attribute "href" not allowed on element "button" at this point.
"file:/azbuild/arizona-bootstrap/_site/docs/2.0/components/navbar-off-canvas/index.html":495.7-495.40: error: Attribute "href" not allowed on element "button" at this point.
"file:/azbuild/arizona-bootstrap/_site/docs/2.0/components/navbar-off-canvas/index.html":498.7-498.40: error: Attribute "href" not allowed on element "button" at this point.
"file:/azbuild/arizona-bootstrap/_site/docs/2.0/components/navbar-off-canvas/index.html":523.7-523.84: error: Attribute "href" not allowed on element "button" at this point.
```

**Comment by danahertzberg** [comment]
Could use help with hover styles and dropdown background. But it's close!

**Review by geijutsuka — DISMISSED** [review]

**Comment by danahertzberg** [comment]
Phew! All the styles are complete and ready for review.

Just need to know if we want Arizona Bootstrap to have the JS for the search button applied. If so, I could use some assistance with that. :)

**Comment by trackleft** [comment]
```
"file:/azbuild/arizona-bootstrap/_site/docs/2.0/components/navbar-off-canvas/index.html":479.7-479.255: error: Bad value "button" for attribute "type" on element "a": Subtype missing.
"file:/azbuild/arizona-bootstrap/_site/docs/2.0/components/navbar-off-canvas/index.html":479.7-479.255: error: Element "a" is missing required attribute "href".
"file:/azbuild/arizona-bootstrap/_site/docs/2.0/components/navbar-off-canvas/index.html":501.7-501.151: error: Element "a" is missing one or more of the following attributes: "href", "role".
"file:/azbuild/arizona-bootstrap/_site/docs/2.0/components/navbar-off-canvas/index.html":512.7-512.151: error: Element "a" is missing one or more of the following attributes: "href", "role".
```

**Comment by danahertzberg** [comment]
@trackleft Ah! I'm on it!

**Comment by danahertzberg** [comment]
> ```
> "file:/azbuild/arizona-bootstrap/_site/docs/2.0/components/navbar-off-canvas/index.html":479.7-479.255: error: Bad value "button" for attribute "type" on element "a": Subtype missing.
> "file:/azbuild/arizona-bootstrap/_site/docs/2.0/components/navbar-off-canvas/index.html":479.7-479.255: error: Element "a" is missing required attribute "href".
> "file:/azbuild/arizona-bootstrap/_site/docs/2.0/components/navbar-off-canvas/index.html":501.7-501.151: error: Element "a" is missing one or more of the following attributes: "href", "role".
> "file:/azbuild/arizona-bootstrap/_site/docs/2.0/components/navbar-off-canvas/index.html":512.7-512.151: error: Element "a" is missing one or more of the following attributes: "href", "role".
> ```

These are fixed. Trying to find the cause of the other errors now..

**Comment by trackleft** [comment]
```
✖ Variable $navbar-offcanvas-link-padding is not being used!
✖ Variable $navbar-offcanvas-dropdown-link-padding is not being used!
```

**Comment by danahertzberg** [comment]
Jeez. I'm so bad at reading this output. Thanks again @trackleft !

**Comment by trackleft** [comment]
I have so much practice breaking the CI haha

**Comment by danahertzberg** [comment]
> I have so much practice breaking the CI haha, All I do now is search for "Err"

There were lots of ERR!s I didn't know what to look at. I'm learning though! lol

**Comment by danahertzberg** [comment]
🎊 ALL CHECKS HAVE PASSED 🎉

File: scss/_navbar-offcanvas.scss [hunk 1]
```diff
@@ -66,15 +66,33 @@
       margin: 0;
       border: none;
     }
+    .input-group {
+      input.form-control {
+        height: 60px;
+        padding-left: 20px;
+      }
+      .input-group-append > button.btn-search {
```
Comment (trackleft) [inline scss/_navbar-offcanvas.scss:74]: I was wondering what we were going to do with similar markup on the quickstart search box.  Thanks!

File: site/docs/2.0/components/navbar-off-canvas.md [hunk 2]
```diff
@@ -12,34 +12,39 @@ Adding the `.navbar-offcanvas` class to your `<nav>` will incorporate an off-can
 
 {% capture example %}
 <div class="d-lg-none d-flex justify-content-end">
-  <button data-toggle="offcanvas" type="button" data-target="#navbarOffcanvasDemo" aria-controls="navbarOffcanvasDemo" class="btn btn-red">
-    <span class="material-icons-sharp"> menu </span>
+  <button data-toggle="offcanvas" type="button" data-target="#navbarOffcanvasDemo" aria-controls="navbarOffcanvasDemo" class="btn btn-red btn-menu">
```
Comment (trackleft) [inline site/docs/2.0/components/navbar-off-canvas.md:15]: Should we update on this page as well? [Arizona Header](https://github.com/az-digital/arizona-bootstrap/blob/main/site/docs/2.0/components/arizona-header.md)?

**Comment by trackleft** [comment]
I think the search JS is probably out of scope unless you think this code can work?
```
      $('#navbarOffcanvasDemo').on('opened.az.offcanvasmenu', function (e) {
	      if ($(e.target.ownerDocument.activeElement).attr('id') === 'jsAzSearch') {
          $('#block-az-barrio-offcanvas-searchform input').trigger('focus');
	      }
      });
```

File: site/docs/2.0/components/navbar-off-canvas.md (hunk 2 above)
Comment (danahertzberg) [reply site/docs/2.0/components/navbar-off-canvas.md:15]: Maybe in a separate PR? Would we just need to add the same offcanvas menu html?

Comment (trackleft) [reply site/docs/2.0/components/navbar-off-canvas.md:15]: New ticket here: https://github.com/az-digital/arizona-bootstrap/issues/213

**Review by joeparsons — APPROVED** [review]

**Review by tadean — APPROVED** [review]

**Pull request merged**